db/bench_test.go

```diff
@@ -127,7 +127,7 @@ func BenchmarkNoIndexSave(b *testing.B) {
 
 	b.ResetTimer()
 
-	for i := 0; i < b.N; i++ {
+	for i := 0; i < len(res); i++ {
```

Comment:
Oh, so nest the loops. I'm not familiar with the bench setups

Reply:
Yep. Depending on how much time it takes to benchmark the action, the Go tooling might call it again with a greater `b.N`, so it can calculate a significant per/op time doing `totaltime/b.N`.